src/Util.cc

```diff
@@ -40,20 +40,21 @@ bool CheckRootBaseFolders(string& root, string& base) {
     // check base
     casacore::File base_folder(base);
     if (!(base_folder.exists() && base_folder.isDirectory(true) && base_folder.isReadable() && base_folder.isExecutable())) {
-        spdlog::critical("Invalid base directory, does not exist or is not a readable directory. Exiting carta.");
-        return false;
-    }
-    // absolute path: resolve symlinks, relative paths, env vars e.g. $HOME
-    try {
-        base = base_folder.path().resolvedName(); // fails on root folder /
-    } catch (casacore::AipsError& err) {
+        spdlog::warn("Invalid base directory, use root directory instead.");
```

Comment:
Can you change this to `"Invalid base directory, using the provided root directory instead"`